src/Avalonia.OpenGL/Controls/OpenGlControlBase.cs

```diff
@@ -263,29 +258,9 @@ protected virtual void OnOpenGlLost()
         
         protected abstract void OnOpenGlRender(GlInterface gl, int fb);
 
-        private class OpenGLCustomDrawOperation : ICustomDrawOperation
+        public bool HitTest(Point point)
```

Comment:
```suggestion
        bool ICustomHitTest.HitTest(Point point)
```

Just to remove this method from code auto-completion.